doc/branches.md

```diff
@@ -1 +1,1845 @@
 # Branches Module
+Module | HTTP Method | URL | Description 
+--- | --- | --- | ---
+[Create Branches](#create-branches) | POST | /create-branches | Create Branches API
+[Edit Branches](#edit-branches) | PUT | /edit-branches/{id} | Edit Branches API
+[Delete Branches](#delete-branches) | DELETE | /delete-branches/{id} | Delete Branches API
+[Get All Branches](#get-all-branches) | GET | /get-allbranches | Get All Branches API
+[Get Branches By ID](#get-branches-byid) | GET | /get-branches-byid/{id} | Get Branches By ID API
+[Get Branches By Branch Code](#get-branches-bybranchcode) | GET | /get-branches-bybranchcode/{kode} | Get Branches By Branch Code API
+
+## <a name="create-branches"></a>Create
+
+### Endpoint 
+POST /create-branches
+
+### Database
+![](./branches-layer.png)
+
+### Headers
+Key | Value 
+--- | ---
+Content-Type | application/json
+Accept | application/json
+Email | administrator@gmail.com
+
+### Request Payloads
+Name | Type | Example Value
+--- | --- | ---
+branchcode | string | 12345  
+companycode | string | TES
```

Comment:
remove companyCode, you can grab from companyCode of user login

Reply:
done